src/shared_utilities/drivers.jl

```diff
@@ -194,9 +194,32 @@ struct CoupledRadiativeFluxes{FT} <: AbstractRadiativeDrivers{FT} end
 """
     CoupledAtmosphere{FT} <: AbstractAtmosphericDrivers{FT}
 
-To be used when coupling to an atmosphere model.
-"""
-struct CoupledAtmosphere{FT} <: AbstractAtmosphericDrivers{FT} end
+To be used when coupling to an atmosphere model. Contains fields that
+are used to compute surface fluxes in the coupled setup, since
+these are computed within ClimaLand.
```

Comment:
Maybe just "To be used when coupling to an atmosphere model. Contains fields that
are used to compute surface fluxes in the coupled setup." - the last clause might make people thing CL computes the fluxes (but really the coupler does, using CL functions...?)

Reply:
perhaps describe the behavior when all are Nothing (how this is used)?